include/LightGBM/config.h

```diff
@@ -744,6 +744,14 @@ struct Config {
   // desc = `NDCG <https://en.wikipedia.org/wiki/Discounted_cumulative_gain#Normalized_DCG>`__ and `MAP <https://makarandtapaswi.wordpress.com/2012/07/02/intuition-behind-average-precision-and-map/>`__ evaluation positions, separated by ``,``
   std::vector<int> eval_at;
 
+  // check = >0
+  // desc = used only with ``multi-error`` metric
```

Comment:
`multi-error` -> `multi_error`
Refer to 
https://github.com/microsoft/LightGBM/blob/a66c34407dcb3f94f6d29b0a5098d7ee0e1dd59b/include/LightGBM/config.h#L726